src/plugin.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Generates a fetch operation after applying a series of fetch helpers
+ * derived from plugins (if applicable).
+ * @param apiRef
+ * @param originalFetch
+ * @return {*}
+ */
+export function generateFetch(originalFetch, apiRef) {
+	const pluginsThatFetch = apiRef.configuration.plugins
+		.filter(plugin => typeof plugin.applyToFetch === 'function');
+
+	if (!pluginsThatFetch.length) return originalFetch;
+
+	let newFetchHelper = null;
+	let index = pluginsThatFetch.length;
+
+	while (index--) {
+		const plugin = pluginsThatFetch[index];
+		newFetchHelper = plugin.applyToFetch(newFetchHelper || originalFetch, apiRef);
+		if (typeof newFetchHelper !== 'function') {
+			console.warn('A Restie plugin altered fetching without returning a function. This will likely cause issues', {
```

Comment:
I think you can safely remove `likely` 
The next statement in the console logs will definitely be an exception 😂